src/containers/table/Table.js

```diff
@@ -105,6 +109,13 @@ class TableView extends React.Component {
     this.showAll = this.showAll.bind(this);
   }
 
+  onClearState(e) {
```

Comment:
the name of this method is a bit confusing, maybe onClearTags orsmth would be more informative?